test/lib.bash

```diff
@@ -121,10 +151,12 @@ function run_knative_serving_rolling_upgrade_tests {
   PROBER_PID=$!
 
   if [[ $UPGRADE_SERVERLESS == true ]]; then
-    local serving_version=$(oc get knativeserving knative-serving -n $SERVING_NAMESPACE -o=jsonpath="{.status.version}")
+    local serving_version
```

Comment:
Worth mention that you can stack multiple local variables. I do that in first line of function. Like:
```bash
function example_func {
  local version upgrade target_version

  version="$1"
  upgrade="${SCRIPT_UPGRADE:-true}"
  target_version="$(calculate_target_version "$version" "$upgrade")"
  # [..]
}
```